Makefile

```diff
@@ -1,51 +1,53 @@
+.DEFAULT_GOAL:=help
+SHELL:=/bin/bash
+
 repository=dfedigital/teaching-vacancies
 
-monitoring-plan: ## Validate monitoring changes
-		TF_WORKSPACE=monitoring terraform init -upgrade=true -input=false terraform/monitoring \
-		&& bin/run-in-env -e monitoring -o tf_subshell -- terraform plan -input=false terraform/monitoring
+##@ Query parameter store to display environment variables. Requires AWS credentials
 
-monitoring-apply: ## Apply monitoring changes
-		TF_WORKSPACE=monitoring terraform init -upgrade=true -input=false terraform/monitoring \
-		&& bin/run-in-env -e monitoring -o tf_subshell -- terraform apply -input=false -auto-approve terraform/monitoring
+.PHONY: print-env
+print-env: ## make -s local print-env > .env
+		$(if $(env), , $(error Usage: make <env> print-env))
+		@bin/run-in-env -t /tvs/dev/app -y terraform/workspace-variables/$(env)_app_env.yml \
+			$(local_override) -o env_stdout $(local_filter)
+
+##@ Set environment (all Gov.UK PaaS except `local`)
```

Comment:
Set environment and corresponding configuration?
(all Gov.UK PaaS except `local`) is not really necessary I think